tests/Commands/FactoryMakeCommandTest.php

```diff
@@ -37,7 +37,7 @@ public function it_makes_factory()
     {
         $code = $this->artisan('module:make-factory', ['name' => 'PostFactory', 'module' => 'Blog']);
```

Comment:
Making a variable that isn't used, remove the `$code` variable

Reply:
Didn't touch that code